### DISCUSSION TITLE
Husk Container has a heat sig

### DISCUSSION THREAD

**Discussion by EebleSheembl** [FakeFishGames/Barotrauma#14946] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Husk containers show up on the thermal goggles

### Reproduction steps

1. have husk container
2. wear thermal goggles

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.6.17.0 (Unto the Breach Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/d396c724131098c39d88c2a58798fb29fcdb5b94 (commit in our private repo). Fix coming up in a future update.
